internal/provider/webhook/datasource/schema.go

```diff
@@ -0,0 +1,46 @@
+package datasource
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+
+	"terraform-provider-render/internal/provider/common/validators"
+)
+
+func Schema(ctx context.Context) schema.Schema {
+	return schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
+				Required:    true,
+				Description: "Unique identifier of the webhook",
+			},
+			"name": schema.StringAttribute{
+				Computed:    true,
+				Description: "Name of the webhook",
+				Validators:  []validator.String{validators.StringNotEmpty},
```

Comment:
I don't think we need a validator on the datasource. If this is empty, something went wrong before this point and we should maybe not throw an error on this.

Reply:
Fair point! Removed it